docs/pt/week07/07-1.md

```diff
@@ -0,0 +1,273 @@
+---
+lang: pt
+lang-ref: ch.07-1
+lecturer: Yann LeCun
+title: Modelos baseados em energia (EBM)
+authors: Karanbir Singh Chahal，Meiyi He, Alexander Gao, Weicheng Zhu
```

Comment:
There's a Chinese character here, between `Chahal` and `Meiyi`.
If this happened everywhere, please, send a PR fixing it everywhere.
Thanks!

Reply:
Hum... I believe that is a strange-looking comma :s. FYI, I copied the headers from docs/en/week07/07-1.md and there, at least on my side, I also see that strange looking comma.